codebuild/bin/codebuild-test.sh

```diff
@@ -17,6 +17,7 @@ set -euxo pipefail
 
 PATH=$PWD/build-tools/bin:$PATH
 ROOT=$PWD
+NUM_CPU_THREADS=$(grep -c ^processor /proc/cpuinfo)
```

Comment:
`nproc` works across a few different platforms:
```suggestion
NUM_CPU_THREADS=$(nproc)
```

Reply:
Thanks!